### ISSUE TITLE
DB paths not being resolved properly

### ISSUE THREAD

**Issue by drpatelh** [nf-core/proteinfold#103]
### Description of the bug

When running without `--alphafold2_db` (default) all of the DB paths are printed as `null/` in the Execution log on stdout. Looks like there is an issue in the way these paths need to be resolved. Used v1.0.0 of the pipeline.

## `params.yml`

```
input: 'https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/samplesheet/v1.0/samplesheet.csv'
outdir: 's3://<OUT_DIR>'
use_gpu: true
```

![image](https://github.com/nf-core/proteinfold/assets/23529759/37653444-a732-4fd3-8cfe-4a5105ae2e89)

Will check to see if these databases are published where required to be re-used.

**Comment by drpatelh** [comment]
Should be fixed in the next NF edge release:
https://github.com/nextflow-io/nextflow/issues/4081

For now, it may be worth trying NF `23.04.2`.

**Comment by JoseEspinosa** [comment]
Also reported when running colabfold_webserver and not providing the `--colabfold_db` parameter, see [this](https://nfcore.slack.com/archives/C0351A8S487/p1712248880698989) slack thread.

**Comment by JoseEspinosa** [comment]
This should be addressed in dev now and made available in v1.1.0, thus will close the issue
